department-of-veterans-affairs/caseflow
# Judge Spreadsheet Validation

Connects #5773 

### Description
This PR adds validation for the judge spreadsheet.

### Testing Plan
1) Download the Judge Assignments template, fill it out with fake data
1) In the rails console, run S3Service::store_file(<file_name>, <file_path>, :filepath)
1) Create a schedule period with the following command: 'JudgeSchedulePeriod.new(user: User.first, start_date: 90.days.ago, end_date: 1.days.ago, file_name: <file_name>).save!
1) Run 'SchedulePeriod.first.validate_spreadsheet', and confirm you get the expected result
1) Repeat with various errors in the spreadsheet
